testsuite/features/step_definitions/common_steps.rb

```diff
@@ -78,7 +78,12 @@
     now = Time.now
     current_minute = now.strftime('%H:%M')
     previous_minute = (now - 60).strftime('%H:%M')
-    break if find(:xpath, "//a[contains(text(),'#{event}')]/../..//td[4][contains(text(),'#{current_minute}') or contains(text(),'#{previous_minute}')]/../td[3]/a[1]", wait: 1)
+    begin
+      break if find(:xpath, "//a[contains(text(),'#{event}')]/../..//td[4][contains(text(),'#{current_minute}') or contains(text(),'#{previous_minute}')]/../td[3]/a[1]", wait: 1)
+    rescue Capybara::ElementNotFound
+      # ignored - pending actions cannot be found
+      sleep 1
```

Comment:
I don't see a reason to add an extra second here.
I would just add a warning log and that's all.

Reply:
done